roadmap.asciidoc

```diff
@@ -77,11 +78,15 @@ i.e. patch releases for the current stable minor release (e.g. 1.2.1.Final) are
 * Provide more detailed monitoring information about Debezium's internal state and health via JMX (e.g. to spot loss of DB connection while the connector still is running)
 * Explore publication of change events via reactive data streams (on top of embedded connector)
 * New implementation of the Debezium embedded engine independent of Kafka Connect APIs
+* Read-only incremental snapshot for SQL Server and Oracle
+* Create PoC for implementing sagas (long-running business transactions spanning multiple (micro-)services) using CDC and the outbox pattern
+* Explore and provide building blocks for implementing CQRS architectures based on change data streams
```

Comment:
I'd consider this one as done - as I tried to show in [this blog post](https://debezium.io/blog/2025/11/28/cqrs/), CQRS can be implemented with Debezium easily and IMHO there's no need to create any additional component or tooling to implement CQRS design pattern.